packages/angular/stories/forms.stories.ts

```diff
@@ -9,31 +9,27 @@ import {
 } from '@angular/forms';
 import { type Form } from '@sl-design-system/form';
 import { type Meta, type StoryFn, moduleMetadata } from '@storybook/angular';
-import {
-  CheckboxDirective,
-  CheckboxGroupDirective,
-  RadioGroupDirective,
-  SelectDirective,
-  SwitchDirective,
-  TextAreaDirective,
-  TextFieldDirective
-} from '../src/forms/index';
-import {
-  ButtonBarComponent,
-  ButtonComponent,
-  CheckboxComponent,
-  CheckboxGroupComponent,
-  FormComponent,
-  FormFieldComponent,
-  InlineMessageComponent,
-  RadioComponent,
-  RadioGroupComponent,
-  SelectComponent,
-  SelectOptionComponent,
-  SwitchComponent,
-  TextAreaComponent,
-  TextFieldComponent
-} from '../src/wrappers';
+import { ButtonComponent } from '../src/button/button.component';
+import { ButtonBarComponent } from '../src/button-bar/button-bar.component';
+import { CheckboxGroupComponent } from '../src/checkbox/checkbox-group.component';
+import { CheckboxComponent } from '../src/checkbox/checkbox.component';
+import { FormFieldComponent } from '../src/form/form-field.component';
+import { FormComponent } from '../src/form/form.component';
+import { CheckboxGroupDirective } from '../src/forms/checkbox-group.directive';
+import { CheckboxDirective } from '../src/forms/checkbox.directive';
+import { RadioGroupDirective } from '../src/forms/radio-group.directive';
+import { SelectDirective } from '../src/forms/select.directive';
+import { SwitchDirective } from '../src/forms/switch.directive';
+import { TextAreaDirective } from '../src/forms/text-area.directive';
+import { TextFieldDirective } from '../src/forms/text-field.directive';
+import { InlineMessageComponent } from '../src/inline-message/inline-message.component';
+import { RadioGroupComponent } from '../src/radio-group/radio-group.component';
```

Comment:
Maybe we can also add wrappers for newer components like `progress bar`?

Reply:
The progress bar wrapper exists, it just isn't in this example story.